nautobot/ipam/tests/test_models.py

```diff
@@ -163,8 +163,16 @@ def test_get_db_prep_value_mysql(self):
 class TestPrefix(TestCase):  # TODO change to BaseModelTestCase
```

Comment:
Now that `setUp` is creating several instances, once the natural-key is re-enabled for this model, you should be able to address this TODO:

```suggestion
class TestPrefix(ModelTestCases.BaseModelTestCase):
```